src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PremisesController.kt

```diff
@@ -291,7 +291,7 @@ class PremisesController(
   }
 
   private fun <ResultType> extractResultEntityOrThrow(result: ValidatableActionResult<ResultType>) = when (result) {
-    is ValidatableActionResult.Success -> result.result
+    is ValidatableActionResult.Success -> result.entity
```

Comment:
Could you squash these commits into the others? I'm a big fan of https://github.com/tummychow/git-absorb for this. No worries if it causes a big mess.

Reply:
sure thing, have sorted